models/articles.models.js

```diff
@@ -30,9 +45,12 @@ function getArticles() {
         FROM articles
         LEFT OUTER JOIN comments
         ON articles.article_id = comments.article_id
+        WHERE lower(articles.topic) LIKE $1
```

Comment:
articles.topic is sql injection. Someone could write drop db and you'd lose your entire database